.github/actions/setup/action.yaml

```
@@ -40,6 +40,19 @@ runs:
         # renovate: datasource=github-tags depName=defenseunicorns/uds-cli versioning=semver
         version: v0.27.10
 
+    - name: Install helm
+      shell: bash
+      run: |
+        curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
+        chmod +x get_helm.sh
+        # renovate: datasource=github-tags depName=helm/helm versioning=semver
+        DESIRED_VERSION=v3.17.0 ./get_helm.sh
```

We should pin the version of the script as well, and make sure this latest ideally:
```suggestion
        # renovate: datasource=github-tags depName=helm/helm versioning=semver
        HELM_VERSION=v3.18.4 ./get_helm.sh
        curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/${HELM_VERSION}/scripts/get-helm-3
        chmod +x get_helm.sh
        DESIRED_VERSION=${HELM_VERSION} ./get_helm.sh
```

---

One other note here - I think the runners already have helm installed: https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md#package-management

We may be able to just delete this section.

---

verified it's not required and removed helm install from setup action